tutorials/android_studio/index.html

```diff
@@ -160,14 +159,15 @@ <h1 class="title">Processing for Android</h1>
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     FrameLayout frame = new FrameLayout(this);
-    frame.setId(ViewIdGenerator.getUniqueId());
+    frame.setId(View.generateViewId());
     setContentView(frame, 
       new ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,
                                  ViewGroup.LayoutParams.MATCH_PARENT));
 
     sketch = new Sketch();
     PFragment fragment = new PFragment(sketch);
-    fragment.setView(frame, this);
```

Comment:
This API corresponds to the beta9 of the mode, which should remain unchanged through the final